src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/TaskTransformer.kt

```diff
@@ -45,14 +48,31 @@ class TaskTransformer(
   )
 
   fun transformPlacementApplicationToTask(placementApplication: PlacementApplicationEntity, offenderDetailSummary: OffenderDetailSummary, inmateDetail: InmateDetail) = PlacementApplicationTask(
+    id = placementApplication.id,
     applicationId = placementApplication.application.id,
     person = personTransformer.transformModelToApi(offenderDetailSummary, inmateDetail),
     dueDate = placementApplication.createdAt.plusDays(10).toLocalDate(),
     allocatedToStaffMember = userTransformer.transformJpaToApi(placementApplication.allocatedToUser!!, ServiceName.approvedPremises) as ApprovedPremisesUser,
     status = getPlacementApplicationStatus(placementApplication),
     taskType = TaskType.placementApplication,
+    risks = risksTransformer.transformDomainToApi(placementApplication.application.riskRatings!!, placementApplication.application.crn),
+    placementDates = placementApplication.placementDates.map {
+      PlacementDates(
+        expectedArrival = it.expectedArrival,
+        duration = it.duration,
+      )
+    },
+    releaseType = placementRequestTransformer.getReleaseType(placementApplication.application.releaseType)!!,
+    placementType = getPlacementType(placementApplication.placementType!!),
   )
 
+  private fun getPlacementType(placementType: JpaPlacementType): ApiPlacementType = when {
+    placementType === JpaPlacementType.ROTL -> ApiPlacementType.rotl
```

Comment:
Go for double equals over triple.  In this case though you could move placementType into the `when`:

```
when (placementType) {
   JpaPlacementType.ROTL -> ApiPlacementType.rotl
   etc.
}
```

Reply:
Argh, that's such a bad habit of mine! 😆 I blame Typescript